tensorflow/contrib/learn/python/learn/datasets/synthetic_test.py

```diff
@@ -125,7 +125,7 @@ def test_spirals_replicable(self):
   def test_spirals_typo(self):
     """Test if a typo was made in a variable name such as x_exrta instead of x_extra
 
-       May raise NameError if there is a typo link in pull request ##16500
+       May raise NameError if there is a typo like in pull request ##16500
     """
     synthetic.spirals(3)
```

Comment:
Thanks for the unit test, @cclauss ! That's very helpful. However, i would probably rewrite the test method in a simpler way:

```python
def test_spirals(self):
  synthetic.spirals(3)
```

The point of the test is that you are covering a (previously uncovered) case of `spirals`, which is valuable in itself. It seems a little strange to write a test for a typo :)